lib/liftoff/project_builder.rb

```diff
@@ -14,11 +14,43 @@ def create_project
       end
 
       xcode_project.save
-      xcode_project.generate_scheme
+      create_schemes
     end
 
     private
 
+    def create_schemes
+      xcode_project.generate_scheme
+
+      if @config.schemes
+        @config.schemes.each do |scheme_config|
+          name = name_from_scheme_config(scheme_config)
```

Comment:
I prefer the current way, so `generate_scheme` doesn't need to know about `scheme_config` directly.